pages/sections.tsx

```diff
@@ -102,8 +124,8 @@ export const SectionsPage: NextPage = () => {
               },
               {
                 name: "Wellbeing support",
-                href: hrefForSlug(PageSlugs.Submit),
-                status: idAndResidencyComplete.result ? undefined : "",
+                href: hrefForSlug(PageSlugs.HomeCheck),
+                status: wellbeingComplete.result ? "Completed" : undefined,
```

Comment:
Hang on. I think this should be:

```suggestion
                status: idAndResidencyComplete.result ? wellbeingComplete.result ? "Completed" : undefined : "",
```

It's not the clearest API for this component. It could do with tidying up for better clarity.